### ISSUE TITLE
Перепилить поведение команды CmdSay

### ISSUE THREAD

**Issue by op-hui** [op-hui/mu2ch#14]
Сейчас at_say хук срабатывает только в том случае, если объект находится внутри контекста (объекта), в котором введена команда say

Нужно переопределить команду say так ,что бы она бродкастилась на все объекты, типа Character, в локации

**Comment by copyright2015** [comment]
тоесть ты хочешь отлавливать то что сказал любой из character`ов на локации?

**Comment by op-hui** [comment]
> тоесть ты хочешь отлавливать то что сказал любой из character`ов на локации?

да, например для того, что если игрок сказал в Сычевальне, Мамка на Кухне это могла услышать

**Comment by op-hui** [comment]
Перенес на второй релиз
